test/helpers/data/app-workload-points-helper.js

```diff
@@ -76,7 +77,49 @@ module.exports.insertDependencies = function (inserts) {
       paroms_enabled: 1,
       parom: 31,
       weighting_arms_lic: 32,
-      weighting_arms_comm: 33
+      weighting_arms_comm: 33,
+      is_t2a: false
+    })
+  })
+  .then(function (ids) {
+    inserts.push({table: 'workload_points', id: ids[0]})
+    return knex('workload_points').returning('id').insert({
+      comm_tier_1: 5,
```

Comment:
Could you expose getting this workload_points object as a function `getWorkloadPoints` and reuse it in the WPC helper? You can see an example of this in `app-reductions-helper`